content/_data/upgrades/2-235-4.adoc

```diff
@@ -0,0 +1,26 @@
+==== Repository Signing Key Update
```

Comment:
This file is redundant as we have one page per LTS baseline, these are then just sections on that page. Would rather do perhaps a changelog banner than this.

Reply:
Thanks. Fixed in https://github.com/jenkins-infra/jenkins.io/pull/3586/commits/616a00dcf447c609eef3cec01c7363fb1c9cddc3